:sparkles: Improve descriptions (#14)
zenika-open-source/how-to-promote-my-open-source: README.md

<summary>⭐ Talk about your project to your friends and colleagues to get your first github stars before publishing to social networks</summary>
<p>

> > A minimum stars amount make your project more trustable than a zero star project. Ask people you know to support your project before doing a public announcement on social medias.

Reviewer: ```suggestion
> A minimum amount of stars make your project more trustable than a project with zero star. Ask people you know to support your project before doing a public announcement on social medias.
```

<summary>📃 Write articles that mention your project</summary>
<p>

> > Write articles to talk about your project. Purpose can be the technical stack you used, how your project works, problems you encountered, etc. Publish your articles to [medium](https://medium.com/) or [dev.to](https://dev.to/).

Reviewer: ```suggestion
> Write articles about your project. Purpose can be the technical stack you used, how your project works, problems you encountered, etc. Publish your articles to [medium](https://medium.com/) or [dev.to](https://dev.to/).
```

<summary>🏆 Reward contributors</summary>
<p>

> > Be nice with people that helped you! Some open-source projects like [gatsby](https://github.com/gatsbyjs/gatsby) reward contributors with goodies. If you can't afford that (which is probably the case), do a public post (on twitter or other platforms) about the contribution and mention the author ([here is an example of tweet](https://twitter.com/FranckAbgrall/status/1139470547492978688)). Open a `Contributors` section to your README in order to publicly thanks them (ex: [vuepress contributors section](https://github.com/vuejs/vuepress#code-contributors)). 

Reviewer: ```suggestion
> Be nice with people that helped you! Some open-source projects like [gatsby](https://github.com/gatsbyjs/gatsby) reward contributors with goodies. If you can't afford that, do a public post (on twitter or other platforms) about the contribution and mention the author ([here is an example of tweet](https://twitter.com/FranckAbgrall/status/1139470547492978688)). Open a `Contributors` section in your README to publicly thanks them (ex: [vuepress contributors section](https://github.com/vuejs/vuepress#code-contributors)). 
```

<summary>🔙 Ask for feedback</summary>
<p>

> > User's feedback is the best way to improve your project. They probably have features ideas that can make your project better.

Reviewer: ```suggestion
> User's feedback is the best way to improve your project. They probably have features and ideas that could make your project better.
```
